bin/brew

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Since we're now using `bash` for this file, I think it would be more idiomatic to use `[[` instead of `[` and `test` everywhere.